app/views/documents/forbidden.html.erb

```diff
@@ -3,3 +3,8 @@
 <div class="govuk-body">
   <%= t("documents.forbidden.description") %>
 </div>
+
+<% primary_org = LinkablesService.new("organisation")
+ .by_content_id(@edition.primary_publishing_organisation_id) %>
+
+<%= t("documents.forbidden.owner", primary_org: primary_org["internal_name"]) %>
```

Comment:
This text is coming through as unstyled as it's not within a `.govuk-body`, also wondering what about nil case again.

Reply:
Well spotted! I forgot to check the updated page after I finished grappling with the linkables. Fixed, but same comment about the nil case as above.